packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -134,6 +134,7 @@ class UpgradeCommandRunner {
       );
     }
     recordState(flutterVersion);
+    globals.printStatus('Upgrading Flutter to ${upstreamRevision} from ${flutterVersion.frameworkRevision} ($workingDirectory...)');
```

Comment:
For just a single identifier you don't need the`{}`:

```suggestion
    globals.printStatus('Upgrading Flutter to $upstreamRevision from ${flutterVersion.frameworkRevision} ($workingDirectory...)');
```

Reply:
I'll fix it.